lua/lazyvim/plugins/extras/editor/mini-files.lua

```diff
@@ -45,12 +45,69 @@ return {
       require("mini.files").refresh({ content = { filter = new_filter } })
     end
 
+    local map_split = function(buf_id, lhs, direction, close_on_file)
```

Comment:
This function is modified from the example in the mini.files repo quite a bit. notably, I made it open the selected file after creating the split, and added close_on_file so that we can have `_plus` keybindings. I also made the keybindings configurable via opts.mappings.